pinot-core/src/main/java/org/apache/pinot/core/operator/transform/function/SingleParamMathTransformFunction.java

```diff
@@ -164,7 +164,7 @@ protected void applyMathOperator(double[] values, int length) {
 
   public static class Log2TransformFunction extends SingleParamMathTransformFunction {
     public static final String FUNCTION_NAME = "log2";
-    public static final double LOG_BASE = Math.log(2);
+    public static final double LOG_BASE = 0.6931471805599453d;
```

Comment:
This isn't necessary, the field is already `static final` so is trusted as a constant

Reply:
done.